inc/saiobject.h

```diff
@@ -58,37 +58,43 @@ typedef struct _sai_object_key_t
 /**
```

Comment:
As per my understanding of the current model, only create time we need the switch-id. Later we only need the OID and OID somehow maps to switch-id. If that is the model, for the calls in this file we dont need switch-id as parameter.